### ISSUE TITLE
0.9.0 - Unable to run race due to missing JDK9

### ISSUE THREAD

**Issue by mveitas** [elastic/rally#399]
**Rally version** (get with `esrally --version`): 0.9.0

**Invoked command**:
`esrally --track-path tracks/contact-search/parent-child --distribution-version 6.1.1 --challenge index-with-geo-queries`

**Configuration file (located in `~/.rally/rally.ini`))**:
```
[meta]
config.version = 13

[system]
env.name = local

[node]
root.dir = /Users/matt.veitas/.rally/benchmarks

[runtime]
java.home = /Library/Java/JavaVirtualMachines/jdk1.8.0_151.jdk/Contents/Home

[benchmarks]
local.dataset.cache = ${node:root.dir}/data

[reporting]
datastore.type = in-memory
datastore.host =
datastore.port =
datastore.secure =
datastore.user =
datastore.password =

[tracks]
default.url = https://github.com/elastic/rally-tracks

[teams]
default.url = https://github.com/elastic/rally-teams

[defaults]
preserve_benchmark_candidate = False

[distributions]
release.1.url = https://download.elasticsearch.org/elasticsearch/elasticsearch/elasticsearch-{{VERSION}}.tar.gz
release.2.url = https://download.elasticsearch.org/elasticsearch/release/org/elasticsearch/distribution/tar/elasticsearch/{{VERSION}}/elasticsearch-{{VERSION}}.tar.gz
release.url = https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-{{VERSION}}.tar.gz
release.cache = true
```

**JVM version**: jdk1.8.0_151

**OS version**: MacOS

**Description of the problem including expected versus actual behavior**:
I just upgraded to the latest 0.9.0 version of Rally and attempted to run a race and ran into the error stating that there was no JDK 9 configured. In the command I am running, I am telling rally to use the binary 6.1.1 distribution. I don't have or intend to install JDK 9 on our systems in the near future

**Provide logs (if relevant)**:
`
[ERROR] Cannot race. ('Error on host 127.0.0.1', No JDK 9 is configured. You cannot benchmark source builds of Elasticsearch on this machine. Please install a JDK 9 and reconfigure Rally with esrally configure)
`

**Comment by mveitas** [comment]
To make sure that it wasn't anything specific in my track definition, I also ran a simple `esrally --distribution-version-6.1.1` command which fails.

**Comment by danielmitterdorfer** [comment]
Thanks for the bug report. I could reproduce the problem locally and I am about to release Rally 0.9.1 which will contain a fix for this.

**Comment by danielmitterdorfer** [comment]
@mveitas I've just released 0.9.1. After you've upgraded everything should work fine now.
